app/journey/JourneyController.scala

```diff
@@ -81,7 +81,10 @@ class JourneyController @Inject() (actions:        Actions,
           emailService.maybeSendEmail(newJourney)
           journeyService
             .upsert(newJourney)
-            .map(_ => Ok)
+            .map { _ =>
+              logger.info(s"Journey updated with Pcipal data [journeyId: ${newJourney.journeyId.value}] [paymentStatus:${notification.Status.toString}]")
```

Comment:
If it's not a problem, could you log also `[paymentItemId:${notification.paymentItemId.value}] ` in that please :)

Reply:
nvm, all good